Force disable module_maps as it doesn't support on Termux

--- a/cc/private/toolchain/unix_cc_toolchain_config.bzl
+++ b/cc/private/toolchain/unix_cc_toolchain_config.bzl
@@ -37,7 +37,7 @@ def _target_os_version(ctx):
     return xcode_config.minimum_os_for_platform_type(platform_type)
 
 def layering_check_features(compiler, extra_flags_per_feature, is_macos):
-    if compiler != "clang":
+    if True:
         return []
     return [
         feature(
